enterprise/coderd/prebuilds/reconcile.go

```diff
@@ -261,6 +261,20 @@ func (c *StoreReconciler) ReconcileAll(ctx context.Context) error {
 			return nil
 		}
 
+		// nolint:gocritic // ReconcilePreset needs Prebuilds Orchestrator permissions.
+		prebuildsMembershipCtx := dbauthz.AsPrebuildsOrchestrator(ctx)
+		membershipReconciler := StoreMembershipReconciler{
+			store:    c.store,
+			clock:    c.clock,
+			snapshot: snapshot,
+			userID:   prebuilds.SystemUserID,
+		}
+		err = membershipReconciler.ReconcileAll(prebuildsMembershipCtx)
+		if err != nil {
+			logger.Warn(ctx, "reconcile prebuild membership", slog.Error(err))
```

Comment:
Maybe use `error` log level here? If we return here - it means that reconciliation loop doesn't work at all.